README.md

```diff
@@ -70,6 +70,61 @@ the user needs to provide very minimal configuration for monitoring any type of
 
 Luminaire can also monitor a set of data points over windows of time instead of tracking individual data points. This approach is well-suited for streaming use cases where sustained fluctuations are of greater concern than individual fluctuations. See [anomaly detection for streaming data](https://zillow.github.io/luminaire/tutorial/streaming.html) for detailed information.
 
+## Examples
+
```

Comment:
Also, why only in the readme.md file and not in the main docs file too? https://zillow.github.io/luminaire/index.html

Reply:
Main files already have the examples. The point here is to have a quick idea about the library when someone views the main github page.